sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.ts

```diff
@@ -297,6 +259,104 @@ export class AssessmentSequentialProcessingService {
     ];
   }
 
+  /**
+   * Gets the full-time program year contribution totals for the provided application numbers.
+   * @param applicationNumbers application numbers.
+   * @returns full-time program year contribution totals.
+   */
+  async getFTProgramYearContributionTotals(
```

Comment:
Please refrain from using PT/FT in method names.
Feel free to add as a comment that will explain it.